### ISSUE TITLE
Problems with Internetexplorer 11 or Edge

### ISSUE THREAD

**Issue by Lehmeiers** [Winetricks/winetricks#1582]
How do you do?
I would like to use the program BOM ( Biet-O-Matic ) under Wine and need for it the Internetexplorer 11 or Edge with the current mshtml.dll. 
Is there a possibility to install this later? 
With the current version of Wine I always get the error message that this Windows version is not supported and under Winetricks dr Internetexplorer is only available up to version 8.

With kind regards R.Lehmeier

Translated with www.DeepL.com/Translator (free version)

**Comment by austin987** [comment]
Currently it's not possible, no.

Where do you see that IE11/Edge are needed? I don't see that mentioned on the appdb or upstream bugzilla.

**Comment by Lehmeiers** [comment]
I don't know what appdb or upstream bugzilla is or what it means, but I need it for the program BOM ( https://bom.sniper-tool.de/BOM/Biet-O-Matic_install.exe ) . 
It is a sniper tool for Ebay and has served me faithfully for years. Since a change of the registration on Ebay there have been problems that another programmer has taken care of.

This one ( https://www.schnapper.de/ ) has solved the problem but he needs files from IE 11, including the file mshtml.dll .
Under Windows it worked as well, but since I moved to Linux I have a problem with it. 
The attempt with Wine failed again and again because I could not install a current IE. Neither in 32-, nor in 64-bit.

Translated with www.DeepL.com/Translator (free version)

**Comment by Lehmeiers** [comment]
I don't know what appdb or upstream bugzilla is or what it means, but I need it for the program BOM ( https://bom.sniper-tool.de/BOM/Biet-O-Matic_install.exe ) . 
It is a sniper tool for Ebay and has served me faithfully for years. Since a change of the registration on Ebay there have been problems that another programmer has taken care of.

This one ( https://www.schnapper.de/ ) has solved the problem but he needs files from IE 11, including the file mshtml.dll .
Under Windows it worked as well, but since I moved to Linux I have a problem with it. 
The attempt with Wine failed again and again because I could not install a current IE. Neither in 32-, nor in 64-bit.

Translated with www.DeepL.com/Translator (free version)

**Comment by Lehmeiers** [comment]
I don't know what appdb or upstream bugzilla is or what it means, but I need it for the program BOM ( https://bom.sniper-tool.de/BOM/Biet-O-Matic_install.exe ) . 
It is a sniper tool for Ebay and has served me faithfully for years. Since a change of the registration on Ebay there have been problems that another programmer has taken care of.

This one ( https://www.schnapper.de/ ) has solved the problem but he needs files from IE 11, including the file mshtml.dll .
Under Windows it worked as well, but since I moved to Linux I have a problem with it. 
The attempt with Wine failed again and again because I could not install a current IE. Neither in 32-, nor in 64-bit.

Translated with www.DeepL.com/Translator (free version)

**Comment by Lehmeiers** [comment]
I don't know what appdb or upstream bugzilla is or what it means, but I need it for the program BOM ( https://bom.sniper-tool.de/BOM/Biet-O-Matic_install.exe ) . 
It is a sniper tool for Ebay and has served me faithfully for years. Since a change of the registration on Ebay there have been problems that another programmer has taken care of.

This one ( https://www.schnapper.de/ ) has solved the problem but he needs files from IE 11, including the file mshtml.dll .
Under Windows it worked as well, but since I moved to Linux I have a problem with it. 
The attempt with Wine failed again and again because I could not install a current IE. Neither in 32-, nor in 64-bit.

Translated with www.DeepL.com/Translator (free version)

**Comment by Lehmeiers** [comment]
I don't know what appdb or upstream bugzilla is or what it means, but I need it for the program BOM ( https://bom.sniper-tool.de/BOM/Biet-O-Matic_install.exe ) . 
It is a sniper tool for Ebay and has served me faithfully for years. Since a change of the registration on Ebay there have been problems that another programmer has taken care of.

This one ( https://www.schnapper.de/ ) has solved the problem but he needs files from IE 11, including the file mshtml.dll .
Under Windows it worked as well, but since I moved to Linux I have a problem with it. 
The attempt with Wine failed again and again because I could not install a current IE. Neither in 32-, nor in 64-bit.

Translated with www.DeepL.com/Translator (free version)

**Comment by austin987** [comment]
I'm asking how you know it needs Internet Explorer 11? Is there somewhere on the website that shows that? I'm curious how you know it's needed if you haven't been able to install it.

**Comment by Lehmeiers** [comment]
I recently moved from Win 7 to Linux Manjaro and on Win 7 it worked with IE 11 but not with IE 9.

**Comment by Lehmeiers** [comment]
I recently moved from Win 7 to Linux Manjaro and on Win 7 it worked with IE 11 but not with IE 9.

**Comment by Lehmeiers** [comment]
I recently moved from Win 7 to Linux Manjaro and on Win 7 it worked with IE 11 but not with IE 9.

**Comment by Lehmeiers** [comment]
I recently moved from Win 7 to Linux Manjaro and on Win 7 it worked with IE 11 but not with IE 9.

**Comment by Faunenschnitt** [comment]
Hi, I am not sure if you prefer a new bug for this issue, but byond needs for support of /tg/ based games often ie11. I have made a bugreport in wine thinking it was a bug in wine but it might be as well the missing ie11 support, as I suppose it will take a very long time to get the native wine implementation of mshtml.dll and xmllite.dll good enough.

(from the chat with CM developers: "It needs to use performance.now to get execution time for timing events."
but yeah, if you have a version later than 10, it should work fine)

byond bug
https://bugs.winehq.org/show_bug.cgi?id=50457

To sum up if it would be possible to install ie11 with winetricks, I would be very happy.

**Comment by seisdr** [comment]
I would be nice if someone could provide a wineprefix with internet explorer installed
